usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -315,16 +315,17 @@ def handle(self, *args, **options):
                 "action_date": format_date(row['action_date']),
             }
 
-            transaction_instance = load_data_into_model(
-                Transaction(), row,
+            transaction_dict = load_data_into_model(
```

Comment:
Thanks for renaming this to match the type of object coming back...it's the little things!